sources/packages/backend/apps/db-migrations/src/sql/Student/Create-student-history-table.sql

```diff
@@ -0,0 +1,76 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.students_history (
```

Comment:
Not a blocker. IMO having a primary key for the history table e.g `history_id` will be better unless it is going to hurt the common insert query.

Reply:
I thought about it and decided not to add a primary key or a serial ID because I did not see a benefit.
Some considerations:
- A `history_id` would not be used for relationships.
- A `history_id` as a primary key would also generate an index, which I do not see the benefit.
- This table will never be updated, only insert operations would happen.
- The above-mentioned reasons would add an unnecessary overhead to the history.

Please let me know if the arguments are not accurate or if I am missing something 😉